### PR TITLE
Update PPP - Cheng Hin

### PR DESCRIPTION
**Pull request by bchenghi**

### CODE REVIEW

File: docs/team/bchenghi.md [hunk 1]
```diff
@@ -1,46 +1,60 @@
...
+    * Highlights: This enhancement affected commands that were added after implementation, and will affect commands added in the future. It required discussions with team members who implemented commands that were undoable, and we discussed the method of implementation for those commands.
+    * Credits: *{https://github.com/se-edu/addressbook-level4 was referenced for this feature, but changes were made to fit into the architecture of the application}*
+
+  * Added conditions for duplicate food items.
+    * What it does: This requires users to add a `Food` that do not have the same `Description`, `ExpiryDate` and `Tags` as another `Food`. The app shows an appropriate error message when the user attempts to add a duplicate `Food` or edit an existing `Food` to be a duplicate.
+    * Justification: Previously, a `Person` was a duplicate if it had the same `Name`, and either the same `Email` or `Phone`. This no longer applied to the application after the first iteration as these fields were all updated, and it would have caused confusion to users. The duplicate condition was changed to check for fields that we thought were appropriate fields for identifying food items.
```
Comment (songyiang) [inline docs/team/bchenghi.md]: Need to switch to the context of Food.

**Review by songyiang — CHANGES_REQUESTED** [review]
Just a minor edit required.

File: docs/team/bchenghi.md (hunk 1 above)
Comment (bchenghi) [reply docs/team/bchenghi.md]: Thanks! Changed.

**Review by songyiang — APPROVED** [review]
LGTM

**Pull request merged**